generate/generate_reference_files.py

```diff
@@ -496,19 +496,23 @@ def generate_siaf_pre_flight_reference_files_nircam():
 
     if (not os.path.isfile(wedge_file) or (overwrite_grism_file)):
         # grism parameters,     see WFSS worksheet in EXCEL SIAF
-        grism_parameters = Table.read(os.path.join(JWST_SOURCE_DATA_ROOT, instrument, 'grism_parameters.txt'), format='ascii.basic', delimiter='\t', guess=False)
+        grism_parameters = Table.read(os.path.join(JWST_SOURCE_DATA_ROOT, instrument, 'nircam_siaf_grism_parameters.txt'), format='ascii.basic', delimiter=',', guess=False)
```

Comment:
Can `os.path.join(JWST_SOURCE_DATA_ROOT, instrument, 'nircam_siaf_grism_parameters.txt')` be switched to just use the `grism_file` variable defined above?